src/CommonDBTM.php

```diff
@@ -478,6 +478,7 @@ public function showForm($ID, array $options = [])
         $this->initForm($ID, $options);
         $new_item = static::isNewID($ID);
         $in_modal = (bool) ($_GET['_in_modal'] ?? false);
+        Html::requireJs('clipboard');
```

Comment:
Is it necessary?

Reply:
Fixed in [31028e9](https://github.com/glpi-project/glpi/pull/16431/commits/31028e919c632fa61084eb60444b1eb53d884526)